sql-plugin/src/main/scala/org/apache/spark/sql/rapids/stringFunctions.scala

```diff
@@ -871,7 +871,7 @@ object GpuRegExpUtils {
           case _ => false
         }
         case RegexGroup(_, term) =>
-          isASTEmptyRepetition(term) 
+          isASTEmptyRepetition(term)
```

Comment:
there is a bunch of whitespace change in this file, lets revert and do this in a separate change.